package.json

```diff
@@ -4,13 +4,12 @@
   "description": "OpenFisca - Legislation Explorer",
   "scripts": {
     "build": "webpack --config ./webpack.config.prod.babel.js",
```

Comment:
dotenv/config is not imported here.
However, process.env is called by webpack.
Is that intentional ?

Reply:
No, you're right, it is important that `PATHNAME` can be taken from the `.env` file. Good catch, thanks!